exo-go/features/run/run.feature

```diff
@@ -0,0 +1,66 @@
+Feature: running Exosphere applications
+
+  As an Exosphere developer
+  I want to have an easy way to run an application on my developer machine
+  So that I can test my app locally.
+
+  Rules:
+  - run "exo run" in the directory of your application to run it
+  - this command boots up all the services of the application
+
+
+  Scenario: booting a functioning Exosphere application
+    Given a running "simple" application
+    Then my machine is running the services:
+      | NAME |
+      | web  |
+
+
+  Scenario: booting a complex Exosphere application
+    Given a running "running" application
+    Then my machine is running ExoCom
+    And my machine is running the services:
+      | NAME  |
+      | web   |
+      | users |
+    # And ExoCom uses this routing:
```

Comment:
This is commented out because the `yaml` library in Go cannot read the following section in `docker-compose.yml` because of `>-`
```
SERVICE_ROUTES: >-
   [{"role":"web","receives":["users.listed","users.created"],"sends":["users.list","users.create"]},{"role":"users","receives":["mongo.list","mongo.create"],"sends":["mongo.listed","mongo.created"],"namespace":"mongo"}]
```
since part of `exo-setup` will be ported Go soon in response to #313 and that strip shouldn't be there anymore, I think we can leave this out for now.